index.html

```diff
@@ -4,10 +4,10 @@
     <meta charset="utf-8" />
     <title>MathML Accessiblity API Mappings 1.0</title>
     <script src="https://www.w3.org/scripts/jquery/1.11.3/jquery.min.js"></script>
-    <script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove" defer="defer"></script>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer="defer"></script>
     <script src="common/script/resolveReferences.js" class="remove"></script>
     <script src="common/biblio.js" class="remove" defer="defer"></script>
-
+    <script src="common/script/mapping-tables.js"></script>
```

Comment:
why do we need to move this inclusion here in this PR?

Reply:
It stopped it breaking.